package.json

```diff
@@ -101,7 +102,7 @@
     "ssb-ebt": "^5.5.2",
     "ssb-friend-pub": "^1.0.5",
     "ssb-friends": "^3.1.12",
-    "ssb-invite": "^2.0.3",
+    "ssb-invite": "^2.1.3",
```

Comment:
hey @christianbundy head up this introduced bugs in windows install!
https://github.com/ssbc/ssb-invite/issues/7